activerecord/lib/active_record/migration.rb

```diff
@@ -127,7 +128,13 @@ def initialize(name = nil)
     end
   end
 
-  class PendingMigrationError < MigrationError#:nodoc:
+  class PendingMigrationError < MigrationError #:nodoc:
```

Comment:
I don't see tests for this. Should we try to implement this with proper coverage in a separate commit?

Reply:
Now that we execute actions, seems like a good idea to test it. Maybe we can even tweak the message to hint the presence of the button and that you can click it?

Reply:
Yes, that sounds like a good idea. I think these tweaks should be made separate to the introduction of this feature though, so I'd be happy to see them in separate commits 😁

Reply:
Added the tests for the action dispatching. For now, I have added them alongside the database tasks tests, as they have the setup figured-out already. Will move it to the proper place once I have more focused time on it.